MAUI/Picker/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: About .NET MAUI Picker control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Picker (SfPicker) control, its basic features, and picker functionalities.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Overview of .NET MAUI Picker (SfPicker)
+
+Syncfusion .NET MAUI Picker (SfPicker) control allows you to select an item and visualizes its items inside a popup or a drop-down UI element. It supports multiple-column layout, header, footer, and selection view customizations. Privides customization as a date picker, time picker, date-time picker, country picker, color picker, etc.
+
+## Key features
+
+* **Header view**: The .NET MAUI Picker allows users to add and customize header text.
+
+* **Column header view**: The .NET MAUI Picker supports a customizable column header view to separate headings for each column.
+
+* **Footer view**: Provides validation buttons (OK and Cancel) in the footer view. The footer text and background color can be customized.
+
+* **Selection view**: Selection view is used to show the selected item and it can be customized.
```

Comment:
* **Selection view**: The selection view is used to show the selected item, and it can be customized.

Reply:
Content modified.